proprietary/tokens/src/brand/ams/typography.compact.tokens.json

```diff
@@ -0,0 +1,17 @@
+{
+  "ams": {
```

Comment:
To maintain the history of the file  you can use `git mv`, just renaming it in the browser deletes and add a new file.